keeper-gradle-plugin/src/main/kotlin/com/slack/keeper/KeeperPlugin.kt

```diff
@@ -127,7 +131,22 @@ class KeeperPlugin : Plugin<Project> {
         val androidJarRegularFileProvider = project.layout.file(androidJarFileProvider)
 
         appExtension.testVariants.configureEach {
```

Comment:
nit: the `matching { ... }` API here could be nice to separate the filtering of `TestVariant`s from their configuration. Up to you.

Reply:
I thought about it but it comes at the cost of possibly unwrapping twice, so I just left it as-is